README.md

```diff
@@ -125,6 +125,7 @@ map: {
 | [selectableGroup] | `boolean` | false | no | Allow to select group when groupBy is used |
 | [selectableGroupAsModel] | `boolean` | true | no | Indicates whether to select all children or group itself  |
 | [items] | `Array<any>` | `[]` | yes | Items array |
+| [waitFor] | `boolean` |  `-` | no | You can block the `select()` of new items while saving |
```

Comment:
I still don't agree with the terminology. `ng-select` is not waiting but it is just not doing something 🏖